webapp/src/Entity/Clarification.php

```diff
@@ -98,6 +98,9 @@ class Clarification extends BaseApiEntity implements ExternalRelationshipEntityI
     #[Serializer\Exclude]
     private ?Clarification $in_reply_to = null;
 
+    /**
+     * @var Collection<int, Clarification>
```

Comment:
Will we fail CI if/when someone doesn't add such statements?

Reply:
I believe so. We could disable this rule but it adds a lot of functionality